vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -34,7 +34,12 @@ import (
 
 /*
 #cgo CFLAGS: -I ../../../../esx_service/vmci
-#include "vmci_client.c"
+#cgo windows LDFLAGS: -L. -lvmci_client
+#ifdef _WIN32
+	#include "vmci_client.h"
+#else
+	#include "vmci_client.c"
```

Comment:
this does  not look pretty. Can we at least hide it in vmci_client.h , or vmci_client.c ? Let's keep #ifdefs in less places...